doc/modules/model_evaluation.rst

```diff
@@ -317,6 +317,106 @@ parameter:
     >>> print(cv_results['test_fn'])
     [0 1 2 3 2]
 
+.. _which_scoring_function:
+
+Which scoring function should I use?
+====================================
+
+Before diving into the details of the many scores and metrics, we want
+to give some guidance, inspired by statistical decision theory, on the choice
+of **scoring functions** for **supervised learning**, see [Gneiting2009]_:
+
+  - *Which scoring function should I use?*
+  - *Which scoring function is a good one for my task?*
+
+In a nutshell, if the scoring function is given, e.g. in a kaggle competition
+or in a business context, use that one.
+If you are free to choose, it starts by considering the ultimate goal and application
+of the prediction. It is useful to distinguish two steps:
+
+* Predicting
+* Decision making
+
+**Prediction:**
+Usually, the response variable :math:`Y` is a random variable, in the sense that there
+is *no deterministic* function :math:`Y = g(X)`.
```

Comment:
Just realised that "X" is not defined.

Reply:
It is our standard X, the only X we ever use.

Reply:
I know but I would prefer to still make it explicit at the end of the sentence.